src/store/index.js

```diff
@@ -75,12 +76,14 @@ export default new Vuex.Store({
     api: build({ config: Api.Config, environment: import.meta.env }),
     printers: PrinterList,
     plateMap: PlateMap,
+    instrumentFlowcellLayout: InstrumentFlowcellLayout,
```

Comment:
I think this can be removed if we change the way instruments are referenced in currentRun

Reply:
@harrietc52